### PR TITLE
Readonly name

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Set main name widget in read-only mode when resource is edited
- Fix error with wrong import

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Extensibility/components-form/NameRenderer.js [hunk 1]
```diff
@@ -16,11 +16,13 @@ export function NameRenderer({
 }) {
   const { t: tExt } = useGetTranslation();
   const extraPaths = schema.get('extraPaths') || [];
+  const editMode = schema.get('editMode') || false;
 
   return (
     <K8sNameField
       value={value}
       kind={resource.kind}
+      readOnly={editMode}
```
Comment (mrCherry97) [inline core-ui/src/components/Extensibility/components-form/NameRenderer.js:25]: We don't need `editMode` in schema, if I use `widget: Name` in schema for input, it is disabling input in Create too. 

image

We need that only in `ExtensibilityCreate` and `ResourceDetails`, like we had it in "old" implementation.

**Review by mrCherry97 — DISMISSED** [review]
If we want to allow user to decide what input can be edited, we need to make it in different way in my opinion.

**Review by dariadomagala — CHANGES_REQUESTED** [review]
All examples are in light mode, we should stick to one approach

**Review by grego952 — APPROVED** [review]

**Pull request merged**